client/flutter/lib/pages/question_index.dart

```diff
@@ -113,39 +112,34 @@ class _QuestionTileState extends State<QuestionTile>
   Widget build(BuildContext context) {
     return Container(
       color: Colors.white,
-      child: Column(children: <Widget>[
-        Divider(
-          height: 1,
-        ),
+      child: Stack(children: <Widget>[
+        Divider(height: 1, thickness: 1,),
```

Comment:
ExpansionTile adds a line of thickness: 1 when expanded, with the previous divider it would became thickness  2. Now with the stack it would be always 1 as figma prototype